src/vendor/azure-uamqp-c/deps/azure-c-shared-utility/adapters/tlsio_openssl.c

```diff
@@ -1045,6 +1047,17 @@ static int create_openssl_instance(TLS_IO_INSTANCE* tlsInstance)
                         log_ERR_get_error("Failed creating OpenSSL instance.");
                         result = __FAILURE__;
                     }
+                    else if (SSL_set_tlsext_host_name(tlsInstance->ssl, tlsInstance->hostname) != 1)
```

Comment:
for my own education; this is our own patched version of the shared-utils dep?  we don't just bring over their changes wholesale?  (I can see why we might do that, just want to understand fully and not assume)

Reply:
no, not our own. it's simply a copy paste of the fix from the shared-utils :P